src/main/resources/default/templates/wondergem/page-script.html.pasta

```diff
@@ -3,13 +3,12 @@
     function createConfirmForDangerousLinks(message) {
         $("a.link-danger, a.button-danger-ask").on('click', function (event) {
             try {
-                var clickedLink = event.currentTarget;
                 var modalElement = $("#danger-link-confirm");
-
-                var okayLink = $(clickedLink).attr('href');
                 var submitBtn = modalElement.find(":submit");
+                var that = $(this);
+
                 submitBtn.click(function () {
-                    window.location.href = okayLink;
```

Comment:
Please keep the old functionality and add a new listener with an extra class so old usages of link-danger dont have to be migrated right away